modules/db/variables.tf

```diff
@@ -1,9 +1,36 @@
-variable "app-name" {}
+variable "app_name" {}
 
-variable "id_of_vpc" {}
+variable "vpc_id" {}
 
 variable "master_password" {}
 
 variable "vpc_rds_security_group_ids" {}
 
-variable "both_db_subnets_name" {}
\ No newline at end of file
+variable "db_subnets_name" {}
```

Comment:
please add details about `db_subnets_name ` variable, like `description` and `type`

Reply:
Done.